src/pages/Search/Search.tsx

```diff
@@ -164,9 +191,17 @@ function Search() {
                       t("change")
                     )}
                   </Button>
+                  {i18n.language === "en" ? (
```

Comment:
it's pretty awk to have this container logic outside of the`ShareButton` component - if we need to know the language, it'd be cleaner to keep this logic in the `ShareButton` component and have a prop for `lang`, and just export `ShareButton`